web/tests/stream_list.test.js

```diff
@@ -102,7 +102,7 @@ function create_social_sidebar_row({mock_template}) {
     stream_has_any_unread_mentions = true;
     stream_list.create_sidebar_row(social);
     assert.equal($social_count.text(), "99");
-    assert.equal($social_unread_mention_info.text(), "@");
+    assert.equal($social_unread_mention_info.text(), "never-been-set");
```

Comment:
These assertions don't make sense. There needs to be a check that the new icon representation is visible.